src/test/role.spec.js

```diff
@@ -0,0 +1,168 @@
+import dotenv from 'dotenv';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sgMail from '@sendgrid/mail';
+import sinon from 'sinon';
+import app from '../index';
+import TestHelper from '../utils/testHelper';
+import db from '../models';
+
+dotenv.config();
+chai.use(chaiHttp);
+chai.should();
+
+let send;
+let userToken;
+let adminToken;
+
+const URL_PREFIX = '/api/v1/auth';
+const URL_PREFIX_USER = '/api/v1/user';
+
+describe('Assign User Role', () => {
+  before((done) => {
+    TestHelper.destroyModel('User');
+    done();
+  });
+
+  beforeEach(async () => {
+    send = sinon.stub(sgMail, 'send').resolves({});
+  });
+
+  afterEach(async () => {
+    send.restore();
+  });
+  describe('PATCH /user/assign_role', () => {
+    before(async () => {
+      await db.User.create({
+        first_name: 'super',
```

Comment:
I think you can do this with the TestHelper class instead

Reply:
ok. thanks